Float.Core/Net/OAuthTokens.cs

```diff
@@ -9,27 +9,33 @@ namespace Float.Core.Net
     public sealed class OAuthTokens
     {
         const int ExpireSoonThresholdSeconds = 60 * 10; // 10 minutes
-        readonly DateTime timeCreated;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="OAuthTokens"/> class.
         /// </summary>
         /// <param name="accessToken">Access token.</param>
         /// <param name="refreshToken">Refresh token.</param>
         /// <param name="durationSeconds">Duration the the token is valid for.</param>
-        public OAuthTokens(string accessToken, string refreshToken, int durationSeconds)
+        /// <param name="created">The timestamp this token was created.</param>
+        public OAuthTokens(string accessToken, string refreshToken, int durationSeconds, DateTime? created = null)
         {
             if (string.IsNullOrWhiteSpace(accessToken))
             {
                 throw new InvalidStringArgumentException(nameof(accessToken));
             }
 
-            timeCreated = DateTime.Now;
+            Created = created ?? DateTime.Now;
```

Comment:
@mediabounds Do we really care about created now or should we just set expired?

Reply:
I could go either way. I decided to keep it as created date and duration since the response from the server only includes duration (so our model more closely reflected what we got from the server).